### DISCUSSION TITLE
Question:The padding parameter.

### DISCUSSION THREAD

**Discussion by zeroseven-hash** [Chlumsky/msdf-atlas-gen#84] [General]
If I understand correctly, the current padding parameter is only between glyph and glyph. However, it doesn't seem to be helpful for applying some special effects.  The correct way should be to apply padding when generating glyph's sdf,  which means that the padding area should also contain sdf data.

**Comment by Chlumsky** [comment]
I don't know of a padding parameter. You are looking for the range (emrange or pxrange) parameter for what you are describing.
